core/src/main/java/pt/webdetails/cda/utils/kettle/RowMetaToTableModel.java

```diff
@@ -203,9 +205,20 @@ public String getColumnName( int column ) {
           }
           @Override
           public Object getValueAt( int rowIndex, int columnIndex ) {
-            return rowMetaAndData.get( rowIndex ).getData()[columnIndex];
+            RowMetaAndData rowMetaData = rowMetaAndData.get( rowIndex );
+            Object value = rowMetaData.getData()[ columnIndex ];
+
+            //getting dates as java.sql.Timestamp and not as java.util.Date makes simpler to deal with them
+            //since they get formated as a simpler format when "stringded"
```

Comment:
this seems to be a problem with the exporter using a toString, and seems likely to happen with other datasources; shouldn't this be fixed there with a date formatter?

Reply:
The alternatives are probably more convoluted than this approach...